pom.xml

```diff
@@ -793,13 +793,22 @@ under the License.</licenseText>
         <plugin>
           <groupId>org.apache.maven.plugins</groupId>
           <artifactId>maven-pmd-plugin</artifactId>
-          <dependencies>
-            <dependency>
-              <groupId>net.sourceforge.pmd</groupId>
-              <artifactId>pmd-core</artifactId>
-              <version>7.14.0</version>
-            </dependency>
-          </dependencies>
+          <configuration>
+            <analysisCache>true</analysisCache>
+            <analysisCacheLocation>${maven.multiModuleProjectDirectory}/.pmd/.cache</analysisCacheLocation>
+            <printFailingErrors>true</printFailingErrors>
+            <rulesets>
+              <ruleset>${maven.multiModuleProjectDirectory}/.pmd/ruleset.xml</ruleset>
+            </rulesets>
+          </configuration>
+          <executions>
+            <execution>
+              <goals>
+                <goal>check</goal>
+              </goals>
+              <phase>verify</phase>
```

Comment:
I think specifying the phase is not necessary, as the plugin by default binds to "verify" anyway.

Reply:
thx.